sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusRuleManagerAsyncClient.java

```diff
@@ -143,24 +142,6 @@ public Mono<Void> createRule(String ruleName, CreateRuleOptions options) {
         return createRuleInternal(ruleName, options);
     }
 
-    /**
```

Comment:
Have we shipped this yet? If so, this is a breaking change. Otherwise, it's fine. Same with the rename.

Reply:
Nope this hasn't been shipped yet. Ww should be good here to make these changes

Reply:
I only refer to the .NET SDK, it has this overload, I added it as well.